Sofa/Sofa/League Matches/Event/EventView.swift

```diff
@@ -84,12 +81,22 @@ class EventView: BaseView {
             $0.bottom.equalToSuperview().inset(10)
         }
     }
+
+    func update(with viewModel: EventViewModel) {
+        self.viewModel = viewModel
```

Comment:
Zapravo ti ne treba spremljenja vrijednost viewMOdela u ovom view, tako da možeš to obrisati.